chore: bump azure helm action

seanson opened the pull request:
This bumps the Azure Helm action to v4 to avoid
the deprecation of Node v16 based actions.

Fixes #17

seanson committed:
chore: bump azure helm action

This bumps the Azure Helm action to v4 to avoid
the deprecation of Node v16 based actions.

Fixes #17

d3adb5 commented:
Hey @seanson, thanks for the contribution! It looks like `azure/setup-helm` still doesn't have a convenient `v4` tag. Instead, could you switch to `v4.0.0`, or would you rather wait for them to create a `v4`? It's been only 4 days since the 4.0.0 release, after all.

d3adb5 committed:
fix: use azure/setup-helm ref v4.0.0

Use azure/setup-helm ref v4.0.0 as there is yet to be a v4 release.

This is due to azure/setup-helm#126.

d3adb5 commented:
Added a commit referencing azure/setup-helm#126. I'll keep an eye on it and update this action again when there is a v4.

seanson commented:
Apologies for the delayed response but thanks for catching the `v4` issue!
